openshift/patroni-postgis/docker/contrib/root/usr/share/scripts/patroni/health_check.sh

```diff
@@ -2,6 +2,9 @@
 set -Eeu
 set -o pipefail
 
+# check for disk space. Fails if usage hits above 90%
+# df "${PATRONI_POSTGRESQL_DATA_DIR:-/home/postgres/pgdata}" --output=pcent | tail -n 1 | awk '{if ($1+0 > 90) exit 1; else exit 0;}'
```

Comment:
why is this commented out?

Reply:
I've added an explanation (I was keeping this code in, as that's what the upstream version does here)